### PR TITLE
add Mikaël Létang

### PR DESCRIPTION
**Pull request by mikaelletang**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -16979,6 +16979,55 @@ var data = {
...
+            "en"
+          ]
+        },
+        {
+          "title": "Faire du DDD tactique avec Symfony",
+          "abstract": "Comment organiser son code pour suivre les bonnes pratiques du DDD dans Symfony",
+          "tags": [
+            "DDD",
+            "Symfony",
+            "PHP",
```
Comment (dadoonet) [inline baggers.js]: Remove `,` at the end.

**Review by dadoonet — CHANGES_REQUESTED** [review]
There is a typo which will break the JSON content if I'm not mistaken.

**Comment by mikaelletang** [comment]
my bad - fixed

**Review by dadoonet — APPROVED** [review]

**Comment by dadoonet** [comment]
Welcome!

**Pull request merged**